frontend/src/components/FavoriteButton.scss

```diff
@@ -0,0 +1,12 @@
+.favorite-button {
```

Comment:
Custom and permanent styles probably should follow [BEM](https://getbem.com/)... read up if you have not heard of it before. Let me know if you want to talk about it.